sdk/core/core-amqp/review/core-amqp.api.md

```diff
@@ -28,9 +28,43 @@ export { AccessToken }
 
 export { AmqpMessage }
 
-export { AmqpMessageHeader }
+// @public
+export interface AmqpMessageHeader {
+    deliveryCount?: number;
+    durable?: boolean;
+    firstAcquirer?: boolean;
+    priority?: number;
+    timeToLive?: number;
+}
 
-export { AmqpMessageProperties }
+// @public
+export const AmqpMessageHeader: {
+    toRheaAmqpMessageHeader(props: AmqpMessageHeader): RheaAmqpMessageHeader;
+    fromRheaAmqpMessageHeader(props: RheaAmqpMessageHeader): AmqpMessageHeader;
+};
+
+// @public
+export interface AmqpMessageProperties {
+    absoluteExpiryTime?: number;
+    contentEncoding?: string;
+    contentType?: string;
+    correlationId?: string | number | Buffer;
+    creationTime?: number;
+    groupId?: string;
+    groupSequence?: number;
+    messageId?: string | number | Buffer;
+    replyTo?: string;
+    replyToGroupId?: string;
+    subject?: string;
+    to?: string;
+    userId?: string;
+}
+
+// @public
+export const AmqpMessageProperties: {
+    toRheaAmqpMessageProperties(props: AmqpMessageProperties): RheaAmqpMessageProperties;
+    fromRheaAmqpMessageProperties(props: RheaAmqpMessageProperties): AmqpMessageProperties;
```

Comment:
Do Service Bus or Event Hubs packages use these from/to for headers/properties at all? The one place where I see this used in Service Bus is the new `toAmqpAnnotatedMessage()` which in itself can be moved to `@azure/core-amqp`.

Reply:
None in event-hubs.
For service-bus, only at the spot that you have mentioned.

Reply:
I'll move AMQPAnnotatedMessage

Reply:
Done